discord-bot/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "rootDir": "src",
+        "composite": true,
+        "target": "es2021",
+        "module": "es2022",
```

Comment:
Interesting, I didn't know Node could run with ES modules. I will have to look into this.